src/commands/build/wranglerjs/mod.rs

```diff
@@ -62,6 +62,10 @@ pub fn run_build(project: &Project) -> Result<(), failure::Error> {
     }
 }
 
+fn rust_present() -> bool {
+    which::which("rustc").is_ok()
+}
+
 //setup a build to run wranglerjs, return the command, the ipc temp file, and the bundle
 fn setup_build(project: &Project) -> Result<(Command, PathBuf, Bundle), failure::Error> {
     for tool in &["node", "npm"] {
```

Comment:
I believe you can add `rustc` here and it will check for its exisiance in the env

Reply:
You are right, updated.